RecoVertex/Configuration/python/RecoVertex_phase2_timing_cff.py

```diff
@@ -0,0 +1,106 @@
+trackRefsForJetsBeforeSorting3D = trackRefsForJets.clone(src="trackWithVertexRefSelectorBeforeSorting3D")
+offlinePrimaryVertices3D = sortedPrimaryVertices.clone(vertices="unsortedOfflinePrimaryVertices3D",
+                                                       particles="trackRefsForJetsBeforeSorting3D")
+offlinePrimaryVertices3DWithBS = offlinePrimaryVerticesWithBS.clone(vertices="unsortedOfflinePrimaryVertices3D:WithBS",
+                                                                    particles="trackRefsForJetsBeforeSorting3D")
+
+from SimTracker.TrackerHitAssociation.tpClusterProducer_cfi import tpClusterProducer
+from SimTracker.TrackAssociatorProducers.quickTrackAssociatorByHits_cfi import quickTrackAssociatorByHits
+from SimTracker.TrackAssociation.trackTimeValueMapProducer_cfi import trackTimeValueMapProducer
+from CommonTools.RecoAlgos.TOFPIDProducer_cfi import tofPID
+_phase2_tktiming_vertexrecoTask = cms.Task( vertexrecoTask.copy() ,
+                                            tpClusterProducer ,
+                                            quickTrackAssociatorByHits ,
+                                            trackTimeValueMapProducer ,
+                                            unsortedOfflinePrimaryVertices4DnoPID ,
+                                            trackWithVertexRefSelectorBeforeSorting4DnoPID ,
+                                            trackRefsForJetsBeforeSorting4DnoPID ,
+                                            offlinePrimaryVertices4DnoPID ,
+                                            offlinePrimaryVertices4DnoPIDWithBS,
+                                            tofPID,
+                                            unsortedOfflinePrimaryVertices4Dfastsim,
+                                            trackWithVertexRefSelectorBeforeSorting4Dfastsim ,
+                                            trackRefsForJetsBeforeSorting4Dfastsim ,
+                                            offlinePrimaryVertices4Dfastsim,
+                                            offlinePrimaryVertices4DfastsimWithBS,
+                                            unsortedOfflinePrimaryVertices3D,
+                                            trackWithVertexRefSelectorBeforeSorting3D ,
+                                            trackRefsForJetsBeforeSorting3D,
+                                            offlinePrimaryVertices3D,
+                                            offlinePrimaryVertices3DWithBS,
+                                            )
+
+from Configuration.Eras.Modifier_phase2_timing_cff import phase2_timing
+phase2_timing.toReplaceWith(vertexrecoTask, _phase2_tktiming_vertexrecoTask)
+phase2_timing.toReplaceWith(unsortedOfflinePrimaryVertices, unsortedOfflinePrimaryVertices4D)
+phase2_timing.toReplaceWith(offlinePrimaryVertices, offlinePrimaryVertices4D)
+phase2_timing.toReplaceWith(offlinePrimaryVerticesWithBS, offlinePrimaryVertices4DWithBS)
```

Comment:
```phase2_timing``` is generic for all variants with timing.
Is this replacement fully applicable?
Shouldn't it be changed to phase2_timing_layer ?

Reply:
So phase2_timing without phase2_timing_layer corresponds to the hybrid fast simulation based partly on simulation truth which was used for the MTD Technical Proposal.  Probably this will never be used again going forward, and I don't think we want to maintain it as such.  I'll find a minimal/clean solution here as not to break anything, since there are a few relvals defined which do use it...

Reply:
thanks.
I checked locally that all of the non-```timing_layer``` wfs with timing are failing

```
	Era_Phase2C4_timing	= (Phase2C4, phase2_timing)	24034.11 D28 24834.11 D30 
	Era_Phase2C6_timing	= (Phase2C6, phase2_timing)	26234.11 D31 
	Era_Phase2_timing		= (Phase2, phase2_timing)	20034.11 D17 21234.11 D21 21634.11 D22 22034.11 D23 23234.11 D36 23634.11 D37
												24434.11 D29 
```

Reply:
I've made sure that phase2+timing remains functional on its own for the moment.  We can remove it later (makes sense to do so at the moment where we fully remove the fastsim also from the phase2_timing_layer era, since we're keeping it in parallel now for comparisons)